.nightfalldlp/config.json

```diff
@@ -1,9 +1,41 @@
 {
-    "detectors": [
-        "CREDIT_CARD_NUMBER",
-        "PHONE_NUMBER",
-        "API_KEY",
-        "CRYPTOGRAPHIC_KEY"
+    "conditions": [
+        {
+            "detector": {
+                "detectorType": "NIGHTFALL_DETECTOR",
+                "nightfallDetector": "CREDIT_CARD_NUMBER",
+                "displayName": "cc num"
+            },
+            "minNumFindings": 1,
+            "minConfidence": "VERY_UNLIKELY"
+        },
+        {
+            "detector": {
+                "detectorType": "NIGHTFALL_DETECTOR",
+                "nightfallDetector": "PHONE_NUMBER",
+                "displayName": "phone num"
+            },
+            "minNumFindings": 1,
+            "minConfidence": "VERY_UNLIKELY"
+        },
+        {
+            "detector": {
+                "detectorType": "NIGHTFALL_DETECTOR",
+                "nightfallDetector": "API_KEY",
+                "displayName": "api key"
+            },
+            "minNumFindings": 1,
+            "minConfidence": "VERY_UNLIKELY"
+        },
+        {
+            "detector": {
+                "detectorType": "NIGHTFALL_DETECTOR",
+                "nightfallDetector": "CRYPTOGRAPHIC_KEY",
+                "displayName": "crypto key"
+            },
+            "minNumFindings": 1,
+            "minConfidence": "VERY_UNLIKELY"
+        }
```

Comment:
similar product question:
do we want to support context rule/ token exclusion rule /inline regex stuff

Reply:
it should already be supported with the v2 go client, seems like they're just not included in our own config for testing

Reply:
Related to this: do we still want to support overall token exclusion now that we have exclusion rules per condition? If so, we'd be filtering out tokens in `tokenExclusionList` below at the very end with all findings aggregated (and filtered) from each condition.

File inclusion/exclusion seems ok to leave as is for now since we don't have that functionality elsewhere.

Reply:
Based on our conversation in standup, ^ we're going to continue to support the global token exclusion/inclusion